src/features/projects/FilterTechnologies.jsx

```diff
@@ -58,41 +59,39 @@ function FilterTechnologies({ technologies, onFilterChange }) {
   );
 }
 
-const mobileStyles = css`
-  @media screen and (max-width: 1024px) {
-    min-width: var(--sidebar-width-mob);
-  }
+const Title = styled.div`
+  display: none;
+  height: 7rem;
+  color: var(--color-white);
+  align-items: center;
+  padding: 2rem;
+  border-bottom: 1px solid var(--color-lines);
 
-  @media screen and (max-width: 768px) {
-    position: fixed;
-    background-color: var(--color-primary-3);
-    border-bottom: 1px solid var(--color-lines);
-    width: calc(100% - 30px);
+  @media only screen and (max-width: 1024px) {
+    display: flex;
   }
 `;
 
 const FilterContainer = styled.div`
-  margin-bottom: 20px;
   min-width: var(--sidebar-width);
   border-right: 1px solid var(--color-lines);
   padding: 0;
-  position: relative;
-
-  ${mobileStyles}
 `;
 
 const FilterWrapper = styled.div`
-  position: fixed;
   min-width: var(--sidebar-width);
-
-  ${mobileStyles}
 `;
 
 const FilterHeading = styled.h4`
   color: var(--color-white);
   border-bottom: 1px solid var(--color-lines);
   width: 100%;
   padding: 1rem 1rem 1rem 2rem;
+  height: 4rem;
+
+  @media only screen and (max-width: 1024px) {
+    background-color: var(--color-grey-0);
+  }
 `;
 
 const ButtonDropdown = styled.button`
```

Comment:
For FilterItem change padding to padding : 0 2rem;

Reply:
![image](https://github.com/MonasefDev/My-portfolio/assets/93090438/906abbd5-690f-4187-a2e2-5d81d87bed12)
all the items i put padding-left : 2rem